src/shared/components/event-management/components/no-tasks-available/no-tasks-available.component.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { NoTasksAvailableComponent } from './no-tasks-available.component';
```

Comment:
Minor comment - Actually looks like the lint warning will be repeated for all of these spec files so probably should be addressed

Reply:
Thank you @connorpgpmcelroy, I have now organised the imports using the command Shift + Alt + O.